src/docs/integrations/pagerduty.mdx

```diff
@@ -0,0 +1,41 @@
+---
+title: "PagerDuty Integration"
+---
+
+## Create a PagerDuty app
+1. In PagerDuty, in the top right, click on "Developer mode"
+1. Select "Create New App"
+
+When configuring the app, use the following values:
+
+| Setting                                                                                                | Value                                         |
+| ------------------------------------------------------------------------------------------------------ | --------------------------------------------- |
+| App Name                                                                                               | Sentry                                        |
+| Brief Description                                                                                      | Trigger alerts through Sentry                 |
+| Category                                                                                               | Error tracking                                |
+| Do you intend to publish the app for all PagerDuty users and the app ecosystem?                        | No, I am not interested                                |
+
+## Configure the PagerDuty app
+Once on the integration page in PagerDuty, under Functionality > Events Integration, click on "Add".
```

Comment:
Maybe this could be like "On the resulting configuration page".

and also what about making "Functionality > Events Integration" bold?